static/app/views/organizationIntegrations/integrationRow.tsx

```diff
@@ -41,7 +41,7 @@ type Props = {
    * in the alert.
    */
   resolveText?: string;
-
+  sentryApp?: SentryApp;
```

Comment:
@leeandher the whole point of the organizationRow is to be agnostic about the kind of integration it is (note that we have a plugin argument but actually it's used by multiple types of integrations). Instead of adding a sentryApp prop to this, you can add a prop like customIcon and just render that instead of PluginIcon if there. You can handle the fallback logic more elegantly in renderIntegrationIcon of sentryAppDetailedView.tsx than in PluginIcon.